injector/dns_disruption.go

```diff
@@ -206,7 +207,17 @@ func (i *DNSDisruptionInjector) Clean() error {
 	// Remove the net_cls classid for cgroup v1
 	if !i.config.Cgroup.IsCgroupV2() {
 		if err := i.config.Cgroup.Write("net_cls", "net_cls.classid", "0"); err != nil {
-			return fmt.Errorf("unable to write net_cls classid: %w", err)
+			targetExists, existsErr := i.config.TargetContainer.Runtime().MaybeExists(context.Background(), i.config.TargetContainer.ID())
```

Comment:
Code is 100% duplicated between disruptions (network/dns), could we find a way to have it at a single place? (the whole removal for net_clf classid for cgroup v1)?

Reply:
What's left is so small that that we don't end up with less code, I no longer see the advantage

Reply:
From my POV it's still 10 LOC, and a risk to miss a change if at any point we need to iterate on this.

What I am thinking here, would consist of creating a func in the `injector` package to centralise the IsCgroupV2 logic + Write + error handling. 

It's not entirely Cgroup logic, more our logic, hence `injector`.

Reply:
From my POV it's still 10 LOC, and a risk to miss a change if at any point we need to iterate on this.

What I am thinking here, would consist of creating a func in the `injector` package to centralise the IsCgroupV2 logic + Write + error handling. 

It's not entirely Cgroup logic, more our logic, hence `injector`.